polyA/fill_align_matrix.py

```diff
@@ -62,7 +66,8 @@ def fill_align_matrix(
     >>> subs = ["", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA...............", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA--A..............."]
     >>> strts = [0, 2, 0]
     >>> sub_mat = [{"AA":1, "AT":-1, "TA":-1, "TT":1, "..":0}] * 3
-    >>> m = fill_align_matrix([0.0, 0.1, 0.1], 41, 0, 31, [0, -25, -25], [0, -5, -5], 1.0, subs, chros, strts, sub_mat)
+    >>> back_freqs = [None] * 3
+    >>> m = fill_align_matrix([0.0, 0.1, 0.1], 41, 0, 31, [0, -25, -25], [0, -5, -5], 1.0, subs, chros, strts, sub_mat, back_freqs)
```

Comment:
Let's add a regular unit test suite (see `load_alignments_test.py` in `tests/`) so that we can test with "real" background frequencies in addition to `None` values. Let me know if you want help with that.